### ISSUE TITLE
Request: Clearly different ' and "

### ISSUE THREAD

**Issue by Zireael07** [jacobxperez/lexica-ultralegible#131]
# Feature Request
Please make ' and " easily distinguishable, e.g. one curly and the other straight, or one slanted and the other straight, etc.

Some programming languages distinguish between the two and it's a frequent pain point for me when coding

**Comment by jacobxperez** [comment]
@Zireael07 Definitely, I will take a closer look at this issue and decide the best approach.

**Comment by HaleyHalcyon** [comment]
> Some programming languages distinguish between the two and it's a frequent pain point for me when coding

Why are you not using a code font?

**Comment by Zireael07** [comment]
@HaleyHalcyon Not all code fonts distinguish the two either! And some that do have other legibility issues

**Comment by HaleyHalcyon** [comment]
> @HaleyHalcyon Not all code fonts distinguish the two either! And some that do have other legibility issues

[Fairfax HD](https://www.kreativekorp.com/software/fonts/fairfaxhd/) is one that has a slanted single quote (looks like the acute accent `´`) and a vertical double quote.
